Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -1351,41 +1351,32 @@ public function isMonolithicPhp()
             return false;
         }
 
-        $kinds = $this->findGivenKind(array(T_OPEN_TAG, T_OPEN_TAG_WITH_ECHO, T_INLINE_HTML));
```

Comment:
we are relying more and more on this function, let us not iterate through every single token if there is no need for that